project_task_default_stage/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     'name': 'Project Task Default Stage',
     'summary': 'Recovery default task stage projects from v8',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Project',
     'author': 'Incaser Informatica S.L., '
```

Comment:
Please switch the author to the new company `Tecnativa` instead of `Incaser...`.